server/.env.example

```
@@ -46,6 +46,15 @@ MS_REDIRECT_URI=http://localhost:4000/api/calendar/outlook/callback
 # TOKEN_ENCRYPTION_KEY is unset in non-test environments.
 CALENDAR_ENCRYPTION_KEY=
 TOKEN_ENCRYPTION_KEY=
+#
+# CALENDAR_ENCRYPTION_KEY encrypts Google / Microsoft OAuth access and refresh
+# tokens (server/services/calendarService.js). Optional — falls back to
+# TOKEN_ENCRYPTION_KEY. Any length. With neither key set, connecting a calendar
+# and every sync fail closed with "Calendar encryption key is not configured".
+CALENDAR_ENCRYPTION_KEY=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The CALENDAR_ENCRYPTION_KEY key is duplicated

(DuplicatedKey)